src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewerPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
+
+interface GeneralCellStyle {
+    borderStyle: "solid" | "dashed" | "dotted" | undefined;
+    borderWidth: number;
+    borderLeftWidth: number;
+    borderTopWidth: number;
+    padding: string;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const generalCellStyle: GeneralCellStyle = {
+    borderStyle: "solid",
+    borderWidth: 1,
```

Comment:
Can you make them consistent? Either strings with units or numbers (I don't mind either, but I did `" _ pt"` for mine. 
Just because it might not be clear that numbers default to `pt` as their unit

Reply:
Same for the ones below, thanks :)